sdl_android/src/main/java/com/smartdevicelink/api/SdlManagerListener.java

```diff
@@ -0,0 +1,22 @@
+package com.smartdevicelink.api;
+
+public interface SdlManagerListener {
+
+	/**
+	 * Called when a manager is ready for use or failed setup
+	 * @param success - success or fail
+	 */
+	void onStart(boolean success);
```

Comment:
I think we should leave onStart as always a successful start. If there is an error use the `onError` method. I believe this param was just a leftover from using the generic completion listener.